_layouts/qa.html

```diff
@@ -0,0 +1,26 @@
+---
+layout: default
+color: no-logo program-page
+color: dark-nav
+---
+<div class='keyline-top pad8y qa-page'>
+  <div class='limiter'>
+    <div class='pad4y col12 clearfix prose min-height'>
+      <h1>{{page.title}}</h1>
```

Comment:
minor nit-pick: since this is the layout for q&a pads, and all q&a pads have the prefix _“Questions & Answers - ”_ in the title, why not add it here instead of repeating it for every session in the markdown files?

Reply:
Also changed. Has the major advantage that changing this prefix later on is much easier this way.